microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,631 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+This tutorial is for developers using WebView2 in immersive HoloLens 2 Unity applications.  WebView2 on HoloLens 2 and the WebView plugin for Unity are both in Preview and are subject to change before general availability.
+
+WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
+
+For WebView2-enabled 2D applications on HoloLens 2, see [Get started with WebView2 in WinUI 2 (UWP) apps](./winui2.md).
+
+
+In this tutorial, you:
+
+* Set up your development tools for creating HoloLens 2 Unity apps that use WebView2 to display web content.
+* Install the Mixed Reality Toolkit (MRTK), by using the Mixed Reality Feature Tool.
+* Create an initial Unity project for HoloLens 2 development.
+* Add the Microsoft Mixed Reality WebView plugin for Unity, by using the Mixed Reality Feature Tool.
+* Configure a WebView2 prefab that displays webpage content in your HoloLens 2 app.
+* Learn about WebView2 concepts and interactions in Unity.
+
+
+<!-- ------------------------------ -->
```

Comment:
@nishitha-burman @michaelfarnsworth Pls review the wording, eg the link text says "HoloLens" not "2", b/c destination page title omits "2".  that page sometimes has "2".

Reply:
I think we should mention this higher up in the documentation. Either the first sentence or after the first two sentences "This tutorial is for developers ...general availability." Or maybe incorporate into the "This tutorial is for developers..."

Reply:
Which Step section should this h4 section be moved into?  Should it become an h2 Step section?  (affects numbering of major Steps)

Reply:
moved draft up, improved it, closing.